spec.html

```diff
@@ -1085,10 +1085,10 @@ <h1>Value Notation</h1>
+      <p>From the perspective of this specification, the word “is” is used to compare two values for equality, as in “If _bool_ is *true*, then ...”, and the word “contains” is used to search for a value inside lists using equality comparisons, as in "If _list_ contains a Record _r_ such that _r_.[[Foo]] is *true*, then ...". From the perspective of the ECMAScript language, language values are compared for quality using the SameValue abstract operation and the abstract operations it transitively calls.</p>
+      <p>For specification values, examples of values without identity include, but are not limited to: mathematical values and extended mathematical values; ECMAScript source text, surrogate pairs, Directive Prologues, etc; UTF-16 code units; Unicode code points; enums; abstract operations, including syntax-directed operations, host hooks, etc; and ordered pairs. Examples of specification values with identity include, but are not limited to: any kind of Records, including Property Descriptors, PrivateElements, etc; Parse Nodes; Lists; <emu-xref href="#sec-set-and-relation-specification-type">Sets</emu-xref> and Relations; Abstract Closures; Data Blocks; Private Names; execution contexts and execution context stacks; agent signifiers; and WaiterLists. These lists are not exhaustive.</p>
+      <p>ECMAScript language values without identity are <emu-xref href="#sec-ecmascript-language-types-undefined-type">*undefined*</emu-xref>, <emu-xref href="#sec-ecmascript-language-types-null-type">*null*</emu-xref>, <emu-xref href="#sec-ecmascript-language-types-boolean-type">Booleans</emu-xref>, <emu-xref href="#sec-ecmascript-language-types-string-type">Strings</emu-xref>, <emu-xref href="#sec-ecmascript-language-types-number-type">Numbers</emu-xref>, and <emu-xref href="#sec-ecmascript-language-types-bigint-type">BigInts</emu-xref>. ECMAScript language values with identity are <emu-xref href="#sec-ecmascript-language-types-symbol-type">Symbols</emu-xref> and <emu-xref href="#sec-object-type">Objects</emu-xref>. Note that Symbols produced by <emu-xref href="#sec-symbol.for">Symbol.for</emu-xref> are a special case; they have identity from the perspective of the specification but do not have identity from the perspective of the ECMAScript language. These lists are exhaustive.</p>
```

Comment:
```suggestion
      <p>The ECMAScript language values without identity are <emu-xref href="#sec-ecmascript-language-types-undefined-type">*undefined*</emu-xref>, <emu-xref href="#sec-ecmascript-language-types-null-type">*null*</emu-xref>, <emu-xref href="#sec-ecmascript-language-types-boolean-type">Booleans</emu-xref>, <emu-xref href="#sec-ecmascript-language-types-string-type">Strings</emu-xref>, <emu-xref href="#sec-ecmascript-language-types-number-type">Numbers</emu-xref>, and <emu-xref href="#sec-ecmascript-language-types-bigint-type">BigInts</emu-xref>. The ECMAScript language values with identity are <emu-xref href="#sec-ecmascript-language-types-symbol-type">Symbols</emu-xref> and <emu-xref href="#sec-object-type">Objects</emu-xref>. Note that Symbols produced by <emu-xref href="#sec-symbol.for">Symbol.for</emu-xref> are a special case; they have identity from the perspective of the specification but do not have identity from the perspective of the ECMAScript language.</p>
```
maybe? Could also keep the last sentence.

Reply:
I like your version better, will apply.